src/ui/simulator/windows/proxy/proxysetup.cpp

```diff
@@ -114,6 +121,8 @@ LicenseCouldNotConnectToInternetServer::LicenseCouldNotConnectToInternetServer(w
     subtitlespacer->Add(title, 0, wxLEFT);
     subtitlespacer->AddSpacer(3);
     subtitlespacer->Add(subtitle, 0, wxLEFT);
+    subtitlespacer->AddSpacer(10);
+    subtitlespacer->Add(offline_title, 0, wxLEFT);
```

Comment:
`offline_title` is never deallocated, as well as other variables such as `title`. `subtitle`, etc.

More generally, this function has dozens of memory leaks...

For now, I suggest that you `delete` only `offline_title` and add a TODO note

Reply:
I don't know if this is the case with wxWidget but with Qt you sometimes don't delete created pointer because they become children of another widget. 

The parent is then responsible for deleting the children

Reply:
@flomnes : you're right, there are things to destroy here, and probably in a lot of other files !
I destroyed the pointer you mention, and i will post an issue for this.